src/integration-tests/introspector/introspectTest.sh

```diff
@@ -393,6 +418,65 @@ function deployCreateDomainJobPod() {
   waitForPod $pod_name
 }
 
+#
+# Create the model in image docker image
+#
+function createMII_Image() {
+  trace "Info: Create MII Image"
+
+  mkdir -p ${test_home}/mii/workdir/models || exit 1
+  cp ${SCRIPTPATH}/mii/models/*  ${test_home}/mii/workdir/models || exit 1
+  cd ${test_home}/mii/workdir  || exit 1
+  echo "place holder" > dummy.txt || exit 1
+  zip ${test_home}/mii/workdir/models/archive.zip dummy.txt > /dev/null 2>&1 || exit 1
+
+  (
+  export WORKDIR=${test_home}/mii/workdir  || exit 1
+  export MODEL_IMAGE_TAG=it || exit 1
+  export MODEL_IMAGE_NAME=model-in-image || exit 1
+  export MODEL_IMAGE_BUILD="when-missing"
+
+  docker rmi ${MODEL_IMAGE_NAME}:${MODEL_IMAGE_TAG} --force > /dev/null 2>&1
+
+  trace "Info: Downloading WDT and WIT..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_download.sh \
+   > ${test_home}/miibuild_download.out 2>&1
+
+  if [ $? -ne 0 ] ; then
+    trace "Error: createMII_Image: download tools failed"
+    cat ${test_home}/miibuild_download.out
+    exit 1
+  fi
+
+  trace "Info: Launching WIT to build the image..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_image_model.sh \
+   > ${test_home}/miibuild_image.out  2>&1
+
+  if [ $? -ne 0 ] ; then
+    trace "Error: createMII_Image: build image failed"
+    cat ${test_home}/miibuild_image.out
+    exit 1
+  fi
+  )
```

Comment:
Forgot to tell you to add an '|| exit 1' after the close paren. (An exit in a paren scope only exits the paren scope.)

Reply:
done